src/chapters/02/main.tex

```diff
@@ -39,6 +39,7 @@ \section{Problem}\label{sec:problem}
 another barber's chair and barber.
 
 The mathematical tool used to model this situation is a Markov chain.
+%%% "The mathematical tool we will use to model this situation..." (This reads like it's the only tool to model this situation)
```

Comment:
Yup **but** see #66 where I remove all "we/us" etc... (I've also written a `README.md` there to capture all these thoughts). But yes this sentence should be changed to something like:

```
The mathematical tool seen in this chapter to model this situation is a Markov chain.
```